src/services/models/MediaType.ts

```diff
@@ -108,30 +122,45 @@ export class MediaTypeModel {
             ]),
           );
         } else {
+          const [generatedExample] = generatedExamples;
```

Comment:
Isn't generatedExamples an empty list? We checked length above.

Reply:
no we check if it has more than 1 element in the array, here we are expecting at most 1 element